lib/common/cmn_files.ml

```diff
@@ -0,0 +1,14 @@
+open Core.Std
+open Core_extended.Std
+open Async.Std
+
+exception Cmn_write_error
+
+let write root name contents =
+  let path = Filename.implode [root; name] in
+  try
+    Writer.save path ~contents
+    >>| fun _ ->
```

Comment:
I wonder if it is safer to write `fun ()` to make sure we don't miss any (very unlikely) changes in `Writer.save`